README.md

```diff
@@ -25,12 +25,19 @@ Works in conjunction with [amonguscapture](https://github.com/denverquane/amongu
 Have any questions, concerns, bug reports, or just want to chat? Join the discord at https://discord.gg/ZkqZSWF!
 
 # Requirements
-
-1. This program must be run on a Windows PC. The program **CANNOT** be run directly on mobile phones.
+## For Everyone
+0. **The Capture application only supports the NON-BETA OFFICIAL STEAM release of the game**
+1. This program must be run on a Windows PC, in Heroku, or in a Docker instance. The program **CANNOT** be run directly on mobile phones.
 2. You need a minimum of 12 open emoji slots on your server. The bot uses player emojis to link discord users to in-game player colors; it will add them automatically, but you need at least 12 slots (25 recommended).
 3. You must run the discord bot, and the capture portion (See Easiest installation below) at the same time. Easiest installation covers running the bot
 portion locally, but feel free to use Heroku or Docker or the like to host the bot remotely.
 
+## For remote usage
```

Comment:
I agree that this falls under the heading for "requirements", but I'm worried about adding a lot of text that precedes the video, written instructions, etc. which users ALREADY skip and don't read haha.  It's a tough balance, but I think maybe moving this networking section down to before "Docker" might be better? If my logic makes sense?